Source/items.cpp

```diff
@@ -3885,205 +3885,170 @@ bool DoOil(Player &player, int cii)
 	switch (plidx) {
 	case IPL_TOHIT:
 	case IPL_TOHIT_CURSE:
-		return fmt::format(fmt::runtime(_("chance to hit: {:+d}%")), item._iPLToHit);
+		return fmt::format(fmt::runtime(_("{:+d}% to Chance To Hit")), item._iPLToHit);

```

Comment:
I am not an English native speaker but it feels a little off to me.

Maybe:
`{:+d}% Chance To Hit`?

Reply:
Can do!